### PR TITLE
chore: remove behaviors

### LINKED ISSUE #110: Move behaviors to behavior module
__What should we do?__

Behaviors in this module are legacy and we have a [dedicated behavior module](https://github.com/camunda/camunda-bpmn-js-behaviors) to host all sorts of modeling behaviors.

We should migrate what is left in this module over.

__Why should we do it?__

Have a single place, not multiple places, to host behaviors.

### PR DESCRIPTION
**Pull request by philippfromme**
BREAKING CHANGES

* behaviors moved to https://github.com/camunda/camunda-bpmn-js-behaviors

---

Closes #110
Related to https://github.com/camunda/camunda-bpmn-js-behaviors/pull/5

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,30 +38,6 @@ Perform a complete build of the application via
 npm run all
 ```
 
-
-## [bpmn-js](https://github.com/bpmn-io/bpmn-js) Extension
```
Comment (nikku) [inline README.md:42]: Should we keep a link to the behaviors at least?

File: package.json [hunk 2]
```diff
@@ -57,17 +55,11 @@
     "webpack": "^5.26.3",
     "xsd-schema-validator": "^0.5.0"
   },
-  "dependencies": {
-    "inherits-browser": "0.0.1",
-    "min-dash": "^3.8.1"
-  },
   "peerDependencies": {
```
Comment (nikku) [inline package.json:58]: Still required?

**Review by nikku — CHANGES_REQUESTED** [review]
Please account for review hints.

File: package.json (hunk 2 above)
Comment (philippfromme) [reply package.json:58]: No.

File: README.md (hunk 1 above)
Comment (philippfromme) [reply README.md:42]: How about that: https://github.com/camunda/camunda-bpmn-moddle/pull/112/commits/dc5d2fb063b4ea4e8f8af425fbce883552ca2233

**Review by nikku — COMMENTED** [review]
One last remark. Looks good otherwise!

**Comment by philippfromme** [comment]
> One last remark. Looks good otherwise!

Ehm, which one? I can't see any others. 👀

**Comment by philippfromme** [comment]
@nikku I assume you were referring to the deep linking. Done (cf. https://github.com/camunda/camunda-bpmn-moddle/pull/112/commits/6bc8aff732f980a4e2bfcac3cf2a0d8b33a6a5d1#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R43).

**Review by nikku — APPROVED** [review]

**Pull request merged**